### PR TITLE
Creating first and basic swagger for new RP of ASI (Azure Security In…

### PR DESCRIPTION
**Pull request by nirgfani**
…sight).

This RP is an extension RP of "Microsoft.operationalinsights", and it is coupled to a Log analytics workspace.

This is a first iteration in order to create a simple RP with single endpoint, after it wil work the API will be changed with the actual and full endpoints based on the learnings from this POC.

There is only one endpoint for alertRules that will be changed in the future.
I validate the swagger with autorest azure guidelines.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/securityinsight/preview/2019-01-01/SecurityInsight.json [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+				{
+					"$ref": "#/parameters/ApiVersion"
+				}
+			],
+			"produces": [
+				"application/json"
+			],
+			"responses": {
+				"200": {
+					"description": "OK. Succesfully retrieved operations list.",
```
Comment (nschonni) [inline specification/securityinsight/preview/2019-01-01/SecurityInsight.json]: Succesfully -> Successfully

File: specification/securityinsight/preview/2019-01-01/SecurityInsight.json [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+        "2019-01-01-preview"
+      ],
+      "description": "API version for the operation"
+    },
+	"OperationalInsightsResourceProvider": {
+		"name": "operationalInsightsResourceProvider",
+		"in": "path",
+		"required": true,
+		"type": "string",
+		"description": "The namespace of workspaces resource provider- Microsoft.operationalinsights.",
```
Comment (nschonni) [inline specification/securityinsight/preview/2019-01-01/SecurityInsight.json]: Microsoft.operationalinsights -> Microsoft.OperationalInsights

File: specification/securityinsight/preview/2019-01-01/SecurityInsight.json [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+        "2019-01-01-preview"
+      ],
+      "description": "API version for the operation"
+    },
+	"OperationalInsightsResourceProvider": {
+		"name": "operationalInsightsResourceProvider",
+		"in": "path",
+		"required": true,
+		"type": "string",
+		"description": "The namespace of workspaces resource provider- Microsoft.Operationalinsights.",
```
Comment (nschonni) [inline specification/securityinsight/preview/2019-01-01/SecurityInsight.json]: Think the "I" in "Insights" probably should be capitalized too

Comment (nirgfani) [reply specification/securityinsight/preview/2019-01-01/SecurityInsight.json]: You right, updated

**Review by sarangan12 — CHANGES_REQUESTED** [review]
@nirgfani  Your swagger does not have a readme.md file. The Individual language generators need a readme.md file to generate the SDK. Please add them so I can review the file

**Review by sarangan12 — CHANGES_REQUESTED** [review]
Please check the errors reported in this link https://travis-ci.org/Azure/azure-rest-api-specs/jobs/477310868 and fix it

**Review by sarangan12 — APPROVED** [review]

**Pull request merged**